frontend/src/services/fetch/fetcherHelpers.ts

```diff
@@ -127,42 +100,12 @@ function fetchErrorToNetworkError(
     : new NetworkError(error);
 }
 
-// note that this will pass along filter inputs in order to maintain the state
-// of the page when relaying an error, but anything passed in the body of the request,
-// such as keyword search query will not be included
-function handleNotOkResponse(
```

Comment:
similar to the above, this wasn't doing much, moved the behavior into the fetcher generator